### PR TITLE
[API-196] Add documentation for FEIN collision handling

### PR DESCRIPTION
**Pull request by eschmidt**
### Purpose

Updating the API docs to reflect changes to the way we handle collisions during partner account provisioning.

### CODE REVIEW

**Comment by zp-alfred** [comment]
Accepted by caseyrollins!

File: _posts/2014-08-01-accounts.markdown [hunk 1]
```diff
@@ -84,3 +84,11 @@ account creation.
   "company_id": 7757616923697256
 }
 ```
+
+#### Collision Handling
+
+A collision is defined as either:
```
Comment (austinmckinley) [inline _posts/2014-08-01-accounts.markdown]:
```suggestion
A collision happens when:
```

**Comment by zp-alfred** [comment]
Accepted by austinmckinley!

**Comment by zp-alfred** [comment]
Accepted by austinmckinley!

**Pull request merged**